docs/usage.md

```diff
@@ -69,6 +72,28 @@ Sample files that can be used as illustrative references can be found in this re
 | grade_percentage | float | A value between 0 and 100 that reflects in the course gradebook |
 | time_submitted | int | A Unix timestamp value that reflects when the grade was created (seconds that have elapsed since 00:00:00 UTC on January 1, 1970) |
 
+## `ib_input_instances.csv`
+
+| Column | Type | Notes |
+| - | - | - |
+| id | UUID | A unique UUID value that is assigned to the input interactive |
+| content_id | UUID | Content page UUID |
+| variant | str | Variant name |
+| content | str | Input content |
+| prompt | str | Prompt content |
+
+## `ib_pset_problems.csv`
```

Comment:
Sorry, I missed this in the initial review, but we're missing the `pset_id` column here.